cloudtools/connect.py

```diff
@@ -54,13 +55,63 @@ def main(args):
     with open(os.devnull, 'w') as f:
         check_call(cmd, stdout=f, stderr=f)
 
-    # open Chrome with SOCKS proxy configuration
+    if sys.platform.startswith('linux'):
```

Comment:
Let's use this for any platform that is not OS X. I think this might resolve a long standing issue wherein our one windows user has to hack up the path in here. I think we just need to add `chrome.exe` to the `names`. @ce-carey are you at broad these days? I'd like to try these changes on your laptop.